Tests/DVRTests/SessionUploadTests.swift

```diff
@@ -86,8 +86,7 @@ class SessionUploadTests: XCTestCase {
     }
 
     private func writeDataToFile(_ data: Data, fileName: String) -> URL {
-        let documentsPath = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)[0]
-        let documentsURL = URL(fileURLWithPath: documentsPath, isDirectory: true)
+        let documentsURL = try! FileManager.default.url(for: .documentDirectory, in: .userDomainMask, appropriateFor: nil, create: true)
```

Comment:
I wonder if this should throw instead of crashing... Not a blocker for this PR!

Reply:
yeah, i just figured since it's in the testing suite it didn't matter too much. if you think it's ok for now, i'll leave it.

Reply:
Yeah, this is totally fine! Either way, the test suite would crash anyway.